cmd/xl-v1-healing.go

```diff
@@ -603,8 +604,8 @@ func defaultHealResult(latestXLMeta xlMetaV1, storageDisks []StorageAPI, errs []
 
 	if !latestXLMeta.IsValid() {
 		// Default to most common configuration for erasure blocks.
-		result.ParityBlocks = len(storageDisks) / 2
-		result.DataBlocks = len(storageDisks) / 2
+		result.ParityBlocks = int(math.Floor(float64(len(storageDisks)) / 2.0))
+		result.DataBlocks = int(math.Ceil(float64(len(storageDisks)) / 2.0))
```

Comment:
These don't add up. Running this:

```
func main() {
	for i := 4; i < 17; i++ {
		parity := int(math.Floor(float64(i)) / 2.0)
		data := int(math.Ceil(float64(i)) / 2.0)
		fmt.Println("drives:", i, " - Data:", data, ", parity:", parity)
	}
}
```
Produces this:

```
drives: 4  - Data: 2 , parity: 2
drives: 5  - Data: 2 , parity: 2
drives: 6  - Data: 3 , parity: 3
drives: 7  - Data: 3 , parity: 3
drives: 8  - Data: 4 , parity: 4
drives: 9  - Data: 4 , parity: 4
drives: 10  - Data: 5 , parity: 5
drives: 11  - Data: 5 , parity: 5
drives: 12  - Data: 6 , parity: 6
drives: 13  - Data: 6 , parity: 6
drives: 14  - Data: 7 , parity: 7
drives: 15  - Data: 7 , parity: 7
drives: 16  - Data: 8 , parity: 8
```

Note how `drives: 15  - Data: 7 , parity: 7`, 14 drives used.

Simpler and working: 

```suggestion
		result.ParityBlocks = (len(storageDisks) + 1) / 2
		result.DataBlocks = len(storageDisks) / 2
```